src/sec_edgar_bulker/core/downloader.py

```
@@ -0,0 +1,500 @@
+        """Run the downloader with the current configuration."""
+        try:
+            # Set up logging
+            self.setup_batch_logging()
+            self.logger.info(f"Starting batch {self.batch_id}")
+            
+            # Process submissions
+            await self.process_submissions()
+            
+            self.logger.info(f"Completed batch {self.batch_id}")
+        except Exception as e:
+            self.logger.error(f"Error in batch {self.batch_id}: {e}")
```

Ruff (0.8.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)